src/main/java/org/numenta/nupic/model/Cell.java

```diff
@@ -142,6 +143,18 @@ public String toString() {
     }
 
     /**
+     * Converts a Collection of {@link Cell}s to {@link Column} indexes.
+     *
+     * @param cells             the list of cells to convert
+     *
+     * @return  sorted array of column indices.
+     */
+    public static int[] asSparseSDR(Collection<Cell> cells) {
+        return cells.stream().mapToInt(c -> c.getColumn().getIndex())
+                .sorted().distinct().toArray();
+    }
+
```

Comment:
> An individual Cell doesn't... 

That's my point, really.

> but the class Cell is actually who knows how to convert between Set<Cell> and int[] (SDR), because each Cell knows its Column. 

Cells shouldn't and don't know anything about SDR's. That's why we have the SDR class, and that's where this belongs in my opinion.